web/src/users/add_url.rs

```diff
@@ -0,0 +1,32 @@
+use axum::{
+    extract::{Form, State},
+    response::{IntoResponse, Redirect},
+};
+use db::{urls::add_url, users::User};
+use serde::Deserialize;
+
+use crate::AppState;
+
+#[derive(Deserialize)]
+pub struct ArticleForm {
+    content: String,
+}
+
+#[axum::debug_handler(state = AppState)]
+pub async fn insert_article_handler(
+    State(state): State<AppState>,
+    user: User,
+    Form(form): Form<ArticleForm>,
+) -> impl IntoResponse {
+    let url = form.content;
+    let user_id = user.user_id;
+
+    println!("inserting");
```

Comment:
Instead of using `println` lets start using the `info!` and `debug! and `error!` traits from `tracing`!
https://github.com/tokio-rs/tracing#in-applications

`tracing` is kinda like a logging and more platform for Rust!
For now basically just replace `println!` with `info!` and `eprintln!` with `error!`
but we'll def do more tracing as we get going!

Reply:
That's very helpful for debugging!